src/auth/service/roles/roles.go

```diff
@@ -0,0 +1,51 @@
+package roles
+
+import (
+	"context"
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	"github.com/consensys/quorum-key-manager/src/auth/entities"
+	"sync"
+
+	"github.com/consensys/quorum-key-manager/src/auth"
+	"github.com/consensys/quorum-key-manager/src/infra/log"
+)
+
+type Interactor struct {
```

Comment:
Why did you name it `Interactor` ? isn't this just a InMemoryRoles struct?

Reply:
called Roles